src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/CommandLineArguments.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices
+{
+    public class CommandLineArguments
```

Comment:
I searched for all references to the `Microsoft.CodeAnalysis.CommandLineArguments` type and only the four properties reflected in this class were ever used.